src/network/server_service.rs

```diff
@@ -34,6 +34,9 @@ pub struct GameListResult(pub Result<GameListResponse, Error>);
 #[derive(Debug, Event)]
 pub struct GameJoinResult(pub Result<GameJoinResponse, Error>);
 
+#[derive(Debug, Event)]
+pub type GameReconnectResult = Result<GameReconnectResponse, Error>;
```

Comment:
This needs fixing

Reply:
Locally already done, but not pushed yet. There was something else somewhere that needed fixing